src/main/java/com/emc/mongoose/storage/driver/pravega/EndToEndLogMessage.java

```diff
@@ -0,0 +1,28 @@
+package com.emc.mongoose.storage.driver.pravega;
+
+import com.emc.mongoose.base.logging.LogMessageBase;
+import static java.lang.System.lineSeparator;
+
+public class EndToEndLogMessage
+	extends LogMessageBase {
```

Comment:
Strange formatting - there's a lot more strings that're longer below.

Reply:
Generally we start a new line when reach "extends" keyword